api/app/signals/apps/signals/factories/attachment.py

```diff
@@ -1,21 +1,24 @@
 from factory import DjangoModelFactory, Sequence, SubFactory, post_generation
-from factory.django import ImageField
+from factory.django import FileField, ImageField
 
 from signals.apps.signals.models import Attachment
 
 
-class ImageAttachmentFactory(DjangoModelFactory):
-
+class AttachmentFactory(DjangoModelFactory):
     class Meta:
         model = Attachment
 
     _signal = SubFactory('signals.apps.signals.factories.signal.SignalFactory')
     created_by = Sequence(lambda n: 'veelmelder{}@example.com'.format(n))
-    file = ImageField()  # In reality it's a FileField, but we want to force an image
+    file = FileField()
     is_image = True
 
     @post_generation
     def set_one_to_one_relation(self, create, extracted, **kwargs):
         self.signal = self._signal
         self.is_image = True
```

Comment:
Nit, but here we have an attachment. No need to set the is_image here. So if we want an attachment to be an image I would move this to the `ImageAttachmentFactory`. Not a blocker for now